client/state/notices/middleware.js

```diff
@@ -299,6 +301,10 @@ export const handlers = {
 	[ JETPACK_MODULE_DEACTIVATE_SUCCESS ]: onJetpackModuleActivationActionMessage,
 	[ JETPACK_MODULE_ACTIVATE_FAILURE ]: onJetpackModuleActivationActionMessage,
 	[ JETPACK_MODULE_DEACTIVATE_FAILURE ]: onJetpackModuleActivationActionMessage,
+	[ INVITE_RESEND_REQUEST_FAILURE ]: dispatchError( translate( 'Invitation failed to resend.' ) ),
+	[ INVITE_RESEND_REQUEST_SUCCESS ]: dispatchSuccess(
+		translate( 'Invitation resent successfully.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Invitation resent.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).